metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -2352,6 +2360,10 @@ def emit_datasource(
                     aspect_name=c.UPSTREAM_LINEAGE,
                     aspect=upstream_lineage,
                 )
+                self.report.num_upstream_table_lineage += len(upstream_tables)
```

Comment:
imo having a simple count of the # of tables with an upstream lineage is more useful. it's likely pretty rare that tables have more than one upstream, but this lets us say things like "we have X% lineage coverage" and is more inline with what we report in other sources

```suggestion
                self.report.num_upstreams_with_table_lineage += 1
```

Reply:
> it's likely pretty rare that tables have more than one upstream

A table or view created with a select statement (CTAS) will have all tables in the FROM as upstream lineage, right? in that case I expect that selects with multiple tables in the FROM will be very common.

Anyway, we can easily have both counters and validate our assumptions.

Reply:
addressed in commit c6875e993bb3a574c0b856f3083df88b5cf1c316

Reply:
_Eg._ these are the metrics I got when testing with Acryl's tableau

```
 'num_tables_with_upstream_lineage': 29,
 'num_upstream_table_lineage': 49,
 'num_upstream_fine_grained_lineage': 516,
 'num_upstream_table_skipped_no_columns': 3,
```